package.json

```diff
@@ -1,23 +1,26 @@
 {
-  "name": "nprogress",
+  "name": "@logdna/nprogress",
```

Comment:
you'll need to add a `publishConfig` key to set the access for the scope to `public`
```json

{
  "private": false
,  "publishConfig": {
      "access": "public"
    }
}

Reply:
done